deps/leveldb/leveldb.gyp

```diff
@@ -26,9 +26,13 @@
     ]
   , 'conditions': [
         ['OS == "win"', {
+            'conditions': [
+                ['MSVS_VERSION != "2015" and MSVS_VERSION != "2013"', {
```

Comment:
I can test it tomorrow but won't this break on 2013 as that version needs the 'port/win' folder? 
Also does this now mandate that the `--msvs_version` has to be passed as part of the build? 
If so this will cause issues with down stream libs that depend on this and we should at least have to warn the user that `--msvc_version` is mandatory on windows somewhere in the output?